helm/credentiald/values.yaml

```
@@ -8,7 +8,18 @@ pod:
     id: 1000
   group:
     id: 1000
-Installation:
-  V1:
-    Registry:
-      Domain: quay.io
+
+registry:
+  domain: docker.io
+
+credentiald:
```

```suggestion
proivder:
```

But I'm not super fussed as this project doesn't get much love anyway.